docs/changelog.rst

```diff
@@ -1,6 +1,23 @@
 Release History
 ===============
 
+v0.9.0
+------
+
+**Disclaimer**
+
+All releases in the v0.x.x series are subject to breaking changes from one version to another. After the release of v1.0.0, this project will be subject to `semantic versioning <http://semver.org/>`_.
+
+**New Features and Enhancements**
+
+* Added ``Scorecard Objectives`` to explain how to include objectives in scorecard templates, patient scorecards, and collection scorecards
+* Achieved 100% code coverage
+* Improved behavior of the ``resolve_by_name`` methods to perform in a case insensitive manne.
```

Comment:
case insensitive manne**r**. (missing "r" in "manner") — Also, none of the other release items end in periods, not sure if that matters.

Reply:
Fixed.